test-packages.txt

```diff
@@ -1,19 +1,18 @@
-aeson
-aeson_extra
-cassava
-conduit
```

Comment:
`conduit` isn't working via this test right now. It fails on `mono-traversable`. I'll add an issue as getting CI back on track is more important right now